src/modules/AutoinstConfig.rb

```diff
@@ -578,6 +618,78 @@ def MainHelp
     publish :function => :ShellEscape, :type => "string (string)"
     publish :function => :AutoinstConfig, :type => "void ()"
     publish :function => :MainHelp, :type => "string ()"
+
+    private
+
+    # Reads base product name from the profile
+    #
+    # FIXME: Currently it returns first found product name. It should be no
+    # problem since this section was unused in AY installation so far.
```

Comment:
No, it is used here:
https://github.com/yast/yast-autoinstallation/blob/90fdf50a1f6167b0dcaf20a5a589d693e2b382a1/src/clients/inst_autosetup_upgrade.rb#L352

we should check whether our use is consistent with that, or whether we should rather specify a new profile element.

Hmm, not documented in https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html#CreateProfile.Software :-(

Reply:
yes that is only usage ... implemented by @jsrain a long time ago.

What i wanted to say is that it was not imported in common (AY) installtion workflow.